roles/servicetelemetry/defaults/main.yml

```diff
@@ -84,11 +84,11 @@ servicetelemetry_defaults:
   graphing:
     enabled: false
     grafana:
-      ingress_enabled: false
-      admin_password: secret
-      admin_user: root
+      ingress_enabled: true
       disable_signout_menu: false
-      base_image: docker.io/grafana/grafana:8.1.2
+      base_image: registry.redhat.io/rhel8/grafana:7
```

Comment:
Would like to change this to grafana:9

Reply:
Addressed by https://github.com/infrawatch/service-telemetry-operator/pull/574/commits/28183df3ad5200351d495b7f823a6d0d15dd0897